src/lib/Preview.js

```diff
@@ -1402,8 +1443,17 @@ class Preview extends EventEmitter {
                 id: fileId,
             },
         };
+
         const headers = getHeaders({}, token, sharedLink, sharedLinkPassword);
 
+        if (this.options.enableAdvancedContentInsights) {
+            this.pageTracker.init(this.previewOptions.contentInsightsOptions, { apiHost, headers }, this.file);
```

Comment:
Should this move into the `finishLoading` method in Preview.js?

Reply:
Similar to the disableEventLog question, the initialization of the tracker is tied to the event log, so I would prefer to keep it here, but I could move it to the finishLoading if you think its better